### PR TITLE
Update About page responsive

### PR DESCRIPTION
**Pull request by huy23121994**

### CODE REVIEW

File: src/pages/Static/About.tsx [hunk 1]
```diff
@@ -211,67 +201,64 @@ export default function About() {
...
           <img src={require('../../assets/svg/about_icon_github.jpg')} />
         </div>
         <div>
-          <Text fontSize={18}>Bug Bounty</Text>
+          <Text fontSize={[12, 18]} fontWeight={500}>Bug Bounty</Text>
           <img src={require('../../assets/svg/about_icon_bug_bounty.svg')} />
         </div>
+        <div>
+          <Text fontSize={[12, 18]} fontWeight={500}>Insured by</Text>
+          <img src={require('../../assets/svg/unslashed.svg')} />
```
Comment (piavgh) [inline src/pages/Static/About.tsx:252]: as I remember, we are not allowed to add Unslashed logo previously. Please ask marketing team to see if we can add it now

Comment (huy23121994) [reply src/pages/Static/About.tsx]: Yes, It's removed

**Review by piavgh — APPROVED** [review]

**Review by piavgh — COMMENTED** [review]
Screenshot at Apr 13 9-49-32 AM

Some sections have no padding. Please update them if you have time

File: src/pages/Static/About.tsx [hunk 2]
```diff
@@ -211,67 +201,60 @@ export default function About() {
...
           Github
         </ButtonOutlined>
         <ButtonOutlined
           padding="12px 28px"
           as={ExternalLink}
           href={`https://files.kyber.network/DMM-Feb21.pdf`}
-          style={{ width: 'auto', marginTop: '5px', fontSize: '16px' }}
+          style={{ width: 'auto', fontSize: '16px' }}
         >
           Lightpaper
```
Comment (piavgh) [inline src/pages/Static/About.tsx:222]: https://docs.google.com/presentation/d/1i26J_-0UtSsSQcjWcR7w8QWwZBPttycjwXyR4B5Xa8s/edit#slide=id.gcd084779ec_0_57

Litepaper

**Pull request merged**